kalohq/ui
# Extending star rating component color

I want to be able to specify the colour of the *Stars* in the `StarRating` component.

It also adds `prop-types` check instead on _flow_.

![Screenshot 2019-05-09 at 14 29 05](https://user-images.githubusercontent.com/435399/57457200-fe49f080-7266-11e9-8598-cbb4ce69bcb9.png)

### How to test it
1. check out the branch
1. `npm install`
1. spin up the style guide
1. check that the `star rating` component looks ok
